### DISCUSSION TITLE
Quorum queues, Message TTL

### DISCUSSION THREAD

**Discussion by andrevus** [rabbitmq/rabbitmq-server#3556] [Other] [answered]
We consider change queue type to quorum, but one feature which we use - message ttl - is not currently supported. We use this as a retry mechanism - when consumer can't process message from some reasons, message will go to dlq with ttl and next move back to queue, so I got a two questions.
Wether message ttl feature in quorum queues will be added?
Can we achieve retry mechanizm in other way?

**Comment by michaelklishin** [comment, marked as answer]
Support for either retries or TTL in quorum queues is planned for a future version. We don't make any ETA promises.
